spring-cloud-generator/scripts/generate.sh

```diff
@@ -19,6 +19,7 @@ fi
 SPRING_ROOT_DIR=${SPRING_GENERATOR_DIR}/..
 
 # Reset target folder for generated code
+cd ${SPRING_GENERATOR_DIR}
```

Comment:
Addressing a previous suggestion in https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1956#discussion_r1231547997: this PR contains small changes to `generate.sh` and `generate_steps.sh` to remove assumption that the `SPRING_GENERATOR_DIR` variable is set in individual helper functions